packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -508,6 +526,13 @@ class CodeCommitSource extends GitSource {
 export interface S3SourceProps extends SourceProps {
   readonly bucket: s3.IBucket;
   readonly path: string;
+
+  /**
+   *  The version ID of the object that represents the build input ZIP file to use.
+   *
+   * @default latest
+   */
+  readonly sourceVersion?: string;
```

Comment:
This should be just called `version` in my opinion (it references an S3 version).